README.md

```diff
@@ -61,15 +61,18 @@ const config = {
 
 This configures Livindocs' image service (the same that is used on edit.livingdocs.io). You can of course also specify your own here or change the parameters for image rendering.
 
-5. Use a design
+5. OPTIONAL: Use a design (If none specified, our magazine example design will be used)
+
 ```
 const axios = require('axios')
-const designRes = await request.get('https://api.livingdocs.io/designs/living-times/0.0.11')
+const designRes = await request.get('https://api.livingdocs.io/designs/living-times/0.0.12')
```

Comment:
actually we are at 0.0.14 now

Reply:
0.0.12 is the latest that we got online though
https://api.livingdocs.io/designs/living-times/0.0.14 doesn't exist

Reply:
ah, good you write this. this will be turned off very soon. our new design server is `https://server.livingdocs.io`